org.devloop.Do.json

```diff
@@ -0,0 +1,37 @@
+{
+    "app-id" : "org.devloop.Do",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "42",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "todo",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "env" : {
+            "CARGO_HOME" : "/run/build/rust-flatpak/cargo"
+        }
+    },
+    "modules" : [
+        {
+            "name" : "todo",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "commit": "26f99cfb488b870fc4ddc0523e006f57f88d3796",
+                    "url" : "https://github.com/edfloreshz/do.git"
+                },
+                "build-aux/cargo-sources.json"
```

Comment:
this file is missing

Reply:
Is it? I thought I included it in this [commit](https://github.com/edfloreshz/do/commit/26f99cfb488b870fc4ddc0523e006f57f88d3796)

Reply:
it needs to be in this repository. It's a manifest. This is included in the manifest.

Reply:
Oh, right! Sorry about that